JS/display_article_js.js

```diff
@@ -128,7 +125,7 @@ httpRequest.onreadystatechange = function () {
 // The code take the data from a json file saved on github.
 httpRequest.open(
```

Comment:
you are requesting nothing here ^^

Reply:
I deleted it since I'm not using it anymore.